float64[] cmd
float64[] vel_cmd